src/gui/AutomationPatternView.cpp

```diff
@@ -278,6 +278,7 @@ void AutomationPatternView::paintEvent( QPaintEvent * )
 
 	const float y_scale = max - min;
 	const float h = ( height() - 2 * TCO_BORDER_WIDTH ) / y_scale;
+	const int MidiTpT = MidiTime::ticksPerTact();
```

Comment:
I think what was suggested was to do away with a computation by doing it once, such as:
`const int MidiTpT = ppt / MidiTime::ticksPerTact();`